packages/e2e-tests/specs/change-detection.test.js

```diff
@@ -240,6 +241,7 @@ describe( 'Change detection', () => {
 		await pressKeyWithModifier( 'primary', 'S' );
 
 		await page.type( '.editor-post-title__input', '!' );
+		await page.waitForSelector( '.editor-post-save-draft' );
```

Comment:
What has changed so that this became necessary?

Reply:
It wasn't deterministically waiting enough time. I think it might have relied on a race condition.